minimint-api/Cargo.toml

```diff
@@ -15,6 +15,7 @@ http-types = "2.12.0"
 minimint-derive = { path = "../minimint-derive" }
 rand = "0.6.0"
 secp256k1-zkp = { git = "https://github.com/fedimint/rust-secp256k1-zkp", branch = "minimint", features = [ "use-serde", "bitcoin_hashes", "global-context" ] }
+rocksdb = { git = "https://github.com/yiyuanliu/rust-rocksdb", branch = "transaction", optional = true }
```

Comment:
Let's put our forked dependency in the `[patch.crates-io]` section of the root `Cargo.toml` [like this](https://github.com/fedimint/minimint/blob/52d39f5cdfcb6cd3a0aa9648720d7f3c7e28477b/Cargo.toml#L18).

Let's use our own [fedimint/rust-rocksdb](https://github.com/fedimint/rust-rocksdb/tree/minimint) fork which I just created. The `minimint` branch there should be identical to yiyuanliu's branch.